nautobot/core/ui/object_detail.py

```diff
@@ -483,17 +485,26 @@ def get_extra_context(self, context):
         for listing and adding objects. It also handles field inclusion/exclusion and
         displays the appropriate table title if provided.
         """
-        body_content_table = context[self.table_key]
+        if isinstance(context[self.table_key], tuple):
+            body_content_table_class, body_content_table_queryset = context[self.table_key]
+            body_content_table = body_content_table_class(body_content_table_queryset)
+        else:
+            body_content_table = context[self.table_key]
         request = context["request"]
 
         if self.exclude_fields or self.include_fields:
-            for column in body_content_table.columns:
```

Comment:
The "pk" column is missing from `body_content_table.columns`, so the toggle was not showing up on the `AssociatedContactsTable` in the contacts tab. This change fixes it.

Reply:
For most related object tables I think we *don't* want the `pk` column, as we don't usually support bulk operations from this view? Might need to be a special case for the AssociatedContactsTable?

Reply:
(bump)

Reply:
Sorry I missed the comment but yeah the pk column is not showing previously